lib/build/bundler.js

```diff
@@ -170,9 +170,17 @@ exports.Bundler = class {
         //Order the bundles so that the bundle containing the config is processed last.
         let configTargetBundleIndex = this.bundles.findIndex(x => x.config.name == this.loaderOptions.configTarget);
         this.bundles.splice(this.bundles.length, 0, this.bundles.splice(configTargetBundleIndex, 1)[0]);
-        return Promise.all(this.bundles.map(x => x.write(this.project.build.targets[0])));
+        if (postBuild && typeof postBuild == 'function') {
+          //If we want to take a different action other than writing to a file, this callback is returned
+          return postBuild();
+        }
+        return this.write();
```

Comment:
This is the one thing I'm not sure about. Would it be better to just remove the call to `write` all together and instead return a promise instead? Then you could simply chain on that to `write` or `generateConfig` depending on who is calling it. Even though it's a internal api (at the moment) I think that would make it cleaner. @AStoker What do you think? Any objections to making that change?